.github/workflows/test.yml

```diff
@@ -58,22 +58,6 @@ jobs:
     - name: test bindata/scripts
       run: make test-bindata-scripts
 
-  modules:
```

Comment:
switch the check to:

1. run go mod tidy
2. check git diff 

WDYT?

Reply:
Good idea. Implemented and pushed